src/anchor/anchor.scss

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-anchor {
+  --_iui-anchor-external-svg: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'><path d='m16 0v5.4l-1.9-2-8.4 8.4-1.5-1.5 8.3-8.4-1.9-1.9m5.4 16v-9h-1v8h-14v-14h8v-1h-9v16z' /></svg>");
+
+  @include iui-anchor-status('primary');
+  border-radius: $iui-border-radius;
+  cursor: pointer;
+
+  &-external::after {
+    content: '';
+    display: inline-block;
+    width: 1.5ch;
+    height: 1.5ch;
+    margin-left: 0.5ch;
+    background-color: currentColor;
+    mask: var(--_iui-anchor-external-svg);
+    @media (forced-colors: active) {
+      background-color: LinkText;
+    }
+  }
+
+  @include iui-anchor-underline('on-hover');
+
+  @media (prefers-contrast: more) {
+    @include iui-anchor-underline('on-default');
+  }
+
+  @at-root {
+    .iui-theme-light &,
+    .iui-theme-dark & {
+      @include iui-anchor-underline('on-hover');
+    }
+  }
+
+  @at-root {
+    .iui-theme-light-hc &,
+    .iui-theme-dark-hc & {
+      @include iui-anchor-underline('on-default');
+    }
+  }
+}
+
+@mixin iui-anchor-status($status: primary) {
```

Comment:
If possible, should we include the different uses of `iui-anchor-status` and `iui-anchor-underline` within the anchor html file?

Reply:
Originally I did have `.iui-positive`, `.iui-warning`, & `.iui-negative` modifier classes so our users could apply any of these status colors.  We removed those classes so they are not exposed to users to try to avoid misuse / abuse of those.  To achieve those status links, you would need to use `@mixin iui-anchor-status`.

That's a long way of explaining that I removed the status links from the anchor demo page because they're already being tested within toast notifications & alerts demo pages.

Reply:
Gotcha, that makes sense.